source/atlas-cli-deploy-docker.txt

```diff
@@ -167,6 +167,124 @@ deployment with `Docker Compose <https://docs.docker.com/compose/>`__.
 
          mongosh mongodb://root:root@localhost:27778/?directConnection=true
 
+Persist Data Across Runs with Docker Compose
+--------------------------------------------
+
+You can persist data across multiple runs with Docker Compose.
```

Comment:
Suggestion: can we mention why they might want to do this/what the advantages are? Does this mean they can see the commands they ran from last time they used the CLI, or something else?

Also, are there prereqs for this, like having the Docker image, etc?

Reply:
@mmarcon, can you confirm the prereqs? I installed Docker and started Docker and installed docker-compose. Anything else?

Reply:
The advantage of persisting data is that it remains available across runs of docker compose. If this is a development environment, for example, it's useful so you don't need to see the cluster with test data every time.

No prereqs, it's all taken care of by having the named docker volumes mounted and the logic inside `entrypoint.sh` does the rest.